src/OpenTracing/Propagation/HttpHeadersCarrier.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http.Headers;
```

Comment:
Is this truly ubiquitous in C#? If not, should we do something more abstract/flexible here? Other OT impls don't *force* the use of a specific HTTP header type.